addons/mail/models/mail_render_mixin.py

```diff
@@ -376,14 +351,14 @@ def _render_template_qweb_view(self, template_src, model, res_ids,
         return results
 
     @api.model
-    def _render_jinja_eval_context(self):
+    def _render_small_qweb_eval_context(self):
         return self._get_common_eval_context()
 
     @api.model
-    def _render_template_jinja(self, template_txt, model, res_ids,
+    def _render_template_small_qweb(self, template_txt, model, res_ids,
```

Comment:
Not sure to understand the purpose of "small qweb". In my mind, we would simply use QWeb everywhere. Content that is not HTML would be enclosed in a tag, rendered, then stripped out of its tag. Content being HTML would be rendered as raw Qweb as before. That way we would not have QWeb and SmallQweb, juste Qweb everywhere that tries to render a "string-like" by enclosing it in a tag then returning its content.

People should not have to choose between two similar options: it should be QWeb everywhere.

What was the rationale behind this choice ?

Reply:
The reason is for the fields `subject`, `email_to`, etc. it is easier and clearer for a user (not a developer) to write `${...}` in the input box rather than `<t t-out="..." />`.

In the future we could have a "smart input text" that will use the wysiwyg editor and provide an easy way to insert fields but we don't have it yet so it has been decided to continue to use the `${...}` syntax.

Reply:
And another subtility is that `SmallQweb` can use `| safe` whereas `<t t-raw=""/>` has been deprecated.

Reply:
> Why is it called "small qweb" when it has nothing whatsoever to do with qweb?

I was waiting the review to find a better name.

> > And another subtility is that SmallQweb can use | safe whereas  has been deprecated.
> 
> As in "it just ignores the filter and never escapes anything" right?

Right.